src/python/py/models/quantized_model.py

```diff
@@ -90,10 +90,11 @@ def __init__(self, quant_type, input_path, bits, group_size, q_size, kv_size, in
         self.lm_head = TensorModule()
         self.layers = []
 
```

Comment:
Should we have `self.layers = {}` instead and then cast it to a list at the end instead of holding two copies (`self.layers` and `layers`)?